All databases actions failing

https://github.com/OrchardCMS/OrchardCore/actions/runs/521424148

These fail to build because of an issue with nuget / debian based operating systems.
```
error NU3028: Package 'System.Collections.Immutable 1.5.0' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain 
```

I think this is due to the use of the  `image: cypress/included:6.0.0` https://github.com/OrchardCMS/OrchardCore/blob/4c71d093a728d726bea24e0024ad9b58a9749aa8/.github/workflows/functional_all_db.yml#L13

This image uses : the `node:12.18.3-buster` image as a base https://github.com/cypress-io/cypress-docker-images/blob/master/base/12.18.3/Dockerfile#L8 

Related to https://github.com/NuGet/Home/issues/10491

--- a/.github/workflows/functional_all_db.yml
+++ b/.github/workflows/functional_all_db.yml
@@ -21,6 +21,9 @@ jobs:
       uses: actions/setup-dotnet@v1
       with:
         dotnet-version: 5.0.102
+    - name: Fix nuget restore issue
+      run: |
+          apt update && apt --only-upgrade install ca-certificates -y
     - name: Functional Tests
       run: |
         cd test/Functional
@@ -48,6 +51,9 @@ jobs:
       uses: actions/setup-dotnet@v1
       with:
         dotnet-version: 5.0.102
+    - name: Fix nuget restore issue
+      run: |
+          apt update && apt --only-upgrade install ca-certificates -y
     - name: Functional Tests
       run: |
         cd test/Functional
@@ -93,6 +99,9 @@ jobs:
       uses: actions/setup-dotnet@v1
       with:
         dotnet-version: 5.0.102
+    - name: Fix nuget restore issue
+      run: |
+          apt update && apt --only-upgrade install ca-certificates -y
     - name: Functional Tests
       run: |
         cd test/Functional
@@ -134,6 +143,9 @@ jobs:
       uses: actions/setup-dotnet@v1
       with:
         dotnet-version: 5.0.102
+    - name: Fix nuget restore issue
+      run: |
+          apt update && apt --only-upgrade install ca-certificates -y
     - name: Functional Tests
       run: |
         cd test/Functional
@@ -174,6 +186,9 @@ jobs:
       uses: actions/setup-dotnet@v1
       with:
         dotnet-version: 5.0.102
+    - name: Fix nuget restore issue
+      run: |
+          apt update && apt --only-upgrade install ca-certificates -y
     - name: Functional Tests
       run: |
         cd test/Functional

--- a/.github/workflows/functional_all_db.yml
+++ b/.github/workflows/functional_all_db.yml
@@ -65,7 +65,7 @@ jobs:
         name: functional-cms-sqlite-failure
         path: |
           test/Functional/cms-tests/cypress/screenshots
-          src/OrchardCore.Cms.Web/App_Data_Tests/logs
+          src/OrchardCore.Cms.Web/App_Data/logs
         retention-days: 3
   
   test_functional_cms_postgresql:
@@ -113,7 +113,7 @@ jobs:
         name: functional-cms-postgresql-failure
         path: |
           test/Functional/cms-tests/cypress/screenshots
-          src/OrchardCore.Cms.Web/App_Data_Tests
+          src/OrchardCore.Cms.Web/App_Data/logs
         retention-days: 3
   
   test_functional_cms_mysql:
@@ -157,7 +157,7 @@ jobs:
         name: functional-cms-mysql-failure
         path: |
           test/Functional/cms-tests/cypress/screenshots
-          src/OrchardCore.Cms.Web/App_Data_Tests
+          src/OrchardCore.Cms.Web/App_Data/logs
         retention-days: 3
 
   test_functional_cms_mssql:
@@ -200,5 +200,5 @@ jobs:
         name: functional-cms-mssql-failure
         path: |
           test/Functional/cms-tests/cypress/screenshots
-          src/OrchardCore.Cms.Web/App_Data_Tests
+          src/OrchardCore.Cms.Web/App_Data/logs
         retention-days: 3

--- a/.github/workflows/functional_all_db.yml
+++ b/.github/workflows/functional_all_db.yml
@@ -113,7 +113,7 @@ jobs:
         name: functional-cms-postgresql-failure
         path: |
           test/Functional/cms-tests/cypress/screenshots
-          src/OrchardCore.Cms.Web/App_Data_Tests/logs
+          src/OrchardCore.Cms.Web/App_Data_Tests
         retention-days: 3
   
   test_functional_cms_mysql:
@@ -157,7 +157,7 @@ jobs:
         name: functional-cms-mysql-failure
         path: |
           test/Functional/cms-tests/cypress/screenshots
-          src/OrchardCore.Cms.Web/App_Data_Tests/logs
+          src/OrchardCore.Cms.Web/App_Data_Tests
         retention-days: 3
 
   test_functional_cms_mssql:
@@ -200,5 +200,5 @@ jobs:
         name: functional-cms-mssql-failure
         path: |
           test/Functional/cms-tests/cypress/screenshots
-          src/OrchardCore.Cms.Web/App_Data_Tests/logs
+          src/OrchardCore.Cms.Web/App_Data_Tests
         retention-days: 3

--- a/.github/workflows/pr_ci.yml
+++ b/.github/workflows/pr_ci.yml
@@ -47,5 +47,5 @@ jobs:
         name: functional-test-failure
         path: |
           test/Functional/cms-tests/cypress/screenshots
-          src/OrchardCore.Cms.Web/App_Data_Tests/logs
+          src/OrchardCore.Cms.Web/App_Data/logs
   

